### PR TITLE
Added code examples for integrating startScreenShare functionality

### PR DESCRIPTION
**Pull request by ktyacke**
Added guide for startScreenShare. I need some folks to review, check spelling/grammar and code samples.

### CODE REVIEW

**Comment by ruffrey** [comment]
:+1:

**Comment by pho3nixf1re** [comment]
:+1:

File: src/js-library/screen-sharing.md [hunk 1]
```diff
@@ -11,15 +11,14 @@ menuOrder: 3
...
 
-*Note: At this time, screen sharing only work in the Chrome browser, and requires a Chrome extension in order to access the screen sharing features (more on how to configure this below). Support for additional browsers will be added in the future.*
+*Note: At this time you must be using the Chrome browser and install an additional Chrome extension in order to use the screen sharing features (more on how to configure this below). The recipient of the Screen Share however, can use any WebRTC enabled browser. Support for additional browsers will be added in the future.*
 
 
 ###Assumptions
-* You have reviewed the [Getting Started With Respoke Guide](https://docs.respoke.io/) 
+* You have reviewed the [Getting Started With Respoke Guide](https://docs.respoke.io/) and are familiar with creating an instance of the `respoke` and `Client` objects.
```
Comment (erinspice) [inline src/js-library/screen-sharing.md]: They don't create `respoke`, just loading the library does that.

File: src/js-library/screen-sharing.md [hunk 2]
```diff
@@ -102,20 +101,57 @@ More info. on inline installation can be found at: [https://developer.chrome.com
...
+client.listen('call', function(evt) {
+    activeCall = evt.call;
 
+    // We only want to answer if we didn't initiate the call
+    if(activeCall.caller !== true) {
+        var videoElement   = document.getElementById('videoElement');
 
+        activeCall.answer({
+            videoLocalElement: videoElement,
+            videoRemoteElement: videoElement
```
Comment (erinspice) [inline src/js-library/screen-sharing.md]: Probably don't want to use the same element here, if just to illustrate that they'll usually be different.

File: src/js-library/screen-sharing.md [hunk 3]
```diff
@@ -102,20 +101,57 @@ More info. on inline installation can be found at: [https://developer.chrome.com
...
+        });
 
+        // The hangup event indicates the call is over
+        activeCall.listen('hangup', function () {
+            activeCall.hangup();
+        });
```
Comment (erinspice) [inline src/js-library/screen-sharing.md]: They don't need to call hangup() inside the `hangup` event listener. If `Call#hangup` gets fired, the call is already hung up. Do something else like remove the video elements from the DOM to illustrate how this event should be used. Or just `console.log("Call got hung up! Show something in the UI to indicate this.")`;

**Comment by erinspice** [comment]
SHIP IT!

**Comment by ruffrey** [comment]
just a reminder, let's delete the branches when merged. :)

**Comment by ktyacke** [comment]
Will do. Meant to ask you about that yesterday.

Thanks.

On Monday, February 2, 2015, Jeff H. Parrish <notifications@github.com>
wrote:

> just a reminder, let's delete the branches when merged. :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/respoke/docs/pull/7#issuecomment-72584509>.
>


-- 
Thanks,

Kyle

**Pull request merged**